projects/docs/docs/contributors/get-started.md

```diff
@@ -52,19 +52,23 @@ Xcode needs to be installed in your system. If not, you can install it from the
 
 ### Edit tuist with tuist
 
-You can also edit tuist with tuist by running `swift run tuist generate` to generate the tuist project with the current changes.
-If you want to use tuist version you have installed in the system, you might want to simply run `tuist generate`.
+You can also edit tuist with tuist by running `./fourier generate tuist` to generate the current tuist project. If you add `--open` flag, it will also automatically open it.
```

Comment:
I think in this doc we are picturing `Edit tuist with tuist` as a secondary way of setting up the project. Shouldn't we push to use the `Project.swift` over the `Package.swift` (and hence move this in the `Set up the project locally` section)?

Reply:
very good point, I think it is related to this PR, updated 👍